lib/Access.php

```diff
@@ -1173,25 +1174,32 @@ private function search($filter, $base, $attr = null, $limit = null, $offset = n
 			list($sr, $pagedSearchOK) = $search;
 			$cr = $this->connection->getConnectionResource();
 
-			if ($skipHandling) {
```

Comment:
this has been moved to `reExecutePagedSearch` as this is only useful when retrying search to get a cache